cmd/kubectl-delivery/Dockerfile

```diff
@@ -1,14 +1,10 @@
-FROM alpine:3.10 AS build
+FROM golang:1.13.6-alpine3.11 AS build
```

Comment:
cc @jtfogarty  @krishnadurai to double check on this

Reply:
@carmark could you please take a look at the guideline PR for creating new docker images?

https://github.com/kubeflow/community/pull/321

Reply:
sure, will do it.